src/window_func.h

```diff
@@ -43,7 +43,7 @@ void DeleteConstructionWindows();
 void HideVitalWindows();
 void ShowVitalWindows();
 
-void ReInitAllWindows();
+void ReInitAllWindows(bool zoom_changed);
```

Comment:
I wonder whether an enum might be more readable here. Currently `ReInitAllWindows(false)` is a bit unclear about what the parameter, and I'd have to go look it up to work out what it was doing

```
enum class ReInitMode {
  Standard,
  ZoomChanged,
}

ReInitAllWindows(ReInitMode::ZoomChanged)
```
Perhaps?

I know the code is full of things like this, but it just occurred to me as I was browsing through

Reply:
I've chosen just to doxygen comment the function, this seems better than adding a single-use enum to me.